docker/entrypoint.sh

```diff
@@ -20,17 +20,23 @@ then
 	exit 1
 fi
 
-ody-stop
```

Comment:
Errr, does it work? I think we need to stop old instance, right?

Reply:
Eithrer way we can remove stop at the end of the file :)

Reply:
This change is ok, ody-stop is called in every test